packages/sysutils/busybox/package.mk

```diff
@@ -251,4 +253,7 @@ makeinstall_init() {
   cp $PKG_DIR/scripts/functions $INSTALL
   cp $PKG_DIR/scripts/init $INSTALL
   chmod 755 $INSTALL/init
+  sed -e "s/@DISTRONAME@/$DISTRONAME/g" \
+        -i $INSTALL/init
```

Comment:
Just a minor: `-i` wants 2 leading spaces removed.

Also, I'd kinda prefer if the `chmod 755 $INSTALL/init` is performed after the `sed`.

And the blank line below isn't required.